includes/amp-helper-functions.php

```diff
@@ -928,14 +928,6 @@ function amp_register_default_scripts( $wp_scripts ) {
 			null
 		);
 	}
-
-	if ( $wp_scripts->query( 'amp-experiment', 'registered' ) ) {
-		/*
-		 * Version 1.0 of amp-experiment is still experimental and requires the user to enable it.
-		 * @todo Revisit once amp-experiment is no longer experimental.
-		 */
-		$wp_scripts->registered['amp-experiment']->src = 'https://cdn.ampproject.org/v0/amp-experiment-0.1.js';
```

Comment:
Note to self: need to update that in Web Stories as well.

Reply:
It turns out that 0.1 is still the `latestVersion`, so you don't have to change it. While 1.0 is valid now, the plugin will continue to serve 0.1 by default.